chef_master/source/knife_windows.rst

```diff
@@ -163,59 +163,104 @@ This argument has the following syntax:
 
 Options
 -----------------------------------------------------
-.. tag knife_windows_bootstrap_windows_ssh_options
 
 This argument has the following options:
 
-``--auth-timeout MINUTES``,
-   The amount of time (in minutes) to wait for authentication to succeed. Default: ``2``.
+``-A``, ``--forward-agent``
+    Enables SSH agent forwarding.
 
-``--bootstrap-no-proxy NO_PROXY_URL_or_IP``
-   A URL or IP address that specifies a location that should not be proxied.
+``--auth-timeout MINUTES``
+    The maximum time in minutes to wait to for authentication over the transport to the node to succeed. Default: ``2``.
+
+``--bootstrap-install-command COMMANDS``
+    Custom command to install chef-client
+
+``--bootstrap-no-proxy [NO_PROXY_URL|NO_PROXY_IP]``
+    A URL or IP address that specifies a location that should not be proxied.
 
 ``--bootstrap-proxy PROXY_URL``
-   The proxy server for the node that is the target of a bootstrap operation.
+    The proxy server for the node that is the target of a bootstrap operation.
+
+``--bootstrap-vault-file VAULT_FILE``
+    A JSON file with a list of vault(s) and item(s) to be updated.
+
+``--bootstrap-vault-item VAULT_ITEM``
+    A single vault and item to update as ``vault:item``.
+
+``--bootstrap-vault-json VAULT_JSON``
+  A JSON string with the vault(s) and item(s) to be updated.
 
 ``--bootstrap-version VERSION``
-   The version of the chef-client to install.
+    The version of the Chef Client to install.
 
 ``-G GATEWAY``, ``--ssh-gateway GATEWAY``
-   The SSH tunnel or gateway that is used to run a bootstrap action on a machine that is not accessible from the workstation.
+    The SSH tunnel or gateway that is used to run a bootstrap action on a machine that is not accessible from the workstation.
 
-``-i IDENTITY_FILE``, ``--identity-file IDENTITY_FILE``
-   The SSH identity file used for authentication. Key-based authentication is recommended.
+``--hint HINT_NAME[=HINT_FILE]``
+    Specify Ohai Hint to be set on the bootstrap targeting multiple nodes. See ``--hint options`` to specify multiple hints.
 
-``-j JSON_ATTRIBS``, ``--json-attributes JSON_ATTRIBS``
-   A JSON string that is added to the first run of a chef-client.
+``--[no]host-key-verify``
+    Use to disable host key verification. Default: ``true``.
```

Comment:
This description assumes that `no-` was specified and it should probably read `Use to enable`